corehq/apps/cloudcare/templates/cloudcare/partials/case_list/tile_grouped_item.html

```diff
@@ -29,12 +29,12 @@
       <% }; %>
     </div>
 
-    <div class="group-rows">
+    <div class="group-rows gap-0">
```

Comment:
doesn't gap affect the gutter between sibling tags? would the one ending on line 30 need this too?

Reply:
they only wanted space between the sub case tiles reduced (which is what this div defines). all other elements were fine with the spacing they already had